src/index.js

```diff
@@ -13,7 +13,8 @@ const {
     deserializeError,
     serializeError,
     extend,
-    emptyTransport
+    emptyTransport,
+    stackTransactionId
```

Comment:
Maybe we should find a more explicit name for this function, like `getTransactionStack` ? @nbeydoon

Reply:
ok